gcp-cloud-run/internal-load-balancer/02-load-balancer.tf

```diff
@@ -65,8 +65,8 @@ resource "google_compute_region_ssl_certificate" "default" {
   region = var.region
   name   = "${var.prefix}-ilb-certificate"
 
-  certificate = file("cert.pem")
-  private_key = file("private_key.pem")
+  certificate = var.ssl_certificate
```

Comment:
will it work with null?

Reply:
it will only be null if the entire module is not deployed (`count = 0`) so it doesnt matter